README-ZH.md

```diff
@@ -0,0 +1,81 @@
+[![Pub](https://img.shields.io/pub/v/flame.svg?style=popout)](https://pub.dartlang.org/packages/flame) ![Test](https://github.com/flame-engine/flame/workflows/Test/badge.svg?branch=master&event=push) [![Discord](https://img.shields.io/discord/509714518008528896.svg)](https://discord.gg/pxrBmy4)
+
+# :fire: flame
+
+<img src="https://i.imgur.com/vFDilXT.png" width="400">
+
+[English](README.md) | 简体中文
+
+一款极简的Flutter游戏引擎.
+
+感激你对我们的任何支持! 评论, 建议, 提issues, Pull Request(贡献&推送)! 给我们点一颗Star, 支持我们! 
+
+## 问题互助
+
+我们在Fireslime的Discord上有一个互助频道, [点击加入](https://discord.gg/pxrBmy4). 现在我们也有了[FAQ](FAQ.md), 所以提问前请先在这里看看你有没有需要的答案.
+
+国内用户欢迎加入[Flame QQ交流群](https://jq.qq.com/?_wv=1027&k=5ETLFm3)(用户不多, 欲入从速)
```

Comment:
Do we want unofficial chat links here?

Reply:
There are very few discord users in China. The main chat rooms in China are WeChat and QQ. I hope to provide a place for Chinese users to communicate. If you mind, I will delete it immediately😃😃

Reply:
I am ok with this been here, as long as we state that this is not an official communication group.

Reply:
Ok, I'll declare that this is an unofficial organization